x-pack/plugins/maps/public/util.ts

```diff
@@ -156,3 +157,37 @@ export async function fetchGeoJson(
     })
   );
 }
+
+export const addFeatureToIndex = async (
+  indexName: string,
+  geometry: Geometry | Position[],
+  path: string
+) => {
+  const data = convertDotNotationStringToObj(path, geometry);
+  return await getHttp().fetch({
+    path: `${INDEX_FEATURE_PATH}`,
+    method: 'POST',
+    body: JSON.stringify({
+      index: indexName,
+      data,
+    }),
+  });
+};
+
+const convertDotNotationStringToObj = (
```

Comment:
Can you add some tests for this function to show what its doing?

Reply:
I actually removed this, we don't need it. Lodash's `set`, or in this case elastic's safe alternative, work perfectly well in its place.